### PR TITLE
3350 overview section update and start score btn

### PR DESCRIPTION
**Pull request by viviancan**
Updating Overview.vue component + all associated wiring up to ProjectDetails for better understanding

### CODE REVIEW

File: app/javascript/judge/scores/sections/ProjectDetails.vue [hunk 1]
```diff
@@ -2,7 +2,7 @@
     <div>
       <GenericJudgingContainer
           heading="Project Name & Description"
-          section="overview"
+          section="project_details"
```
Comment (viviancan) [inline app/javascript/judge/scores/sections/ProjectDetails.vue:5]: The only way I could get the question comment section to render properly was by using snake case as section name!

**Comment by viviancan** [comment]
Updating the naming due to issue where the "title" was undefined on the question component due to mismatched naming. This is now working as expected, however, I am unsure if using snake case in this way is best practice.

**Comment by shaunxp20** [comment]
Maybe this is okay, but I am wondering if the new project details comment gets saved to the database. 🤔 

Also, we should rename the database fields, either in this PR or as a task in #3350, other than that, LGTM!

**Review by shaunxp20 — APPROVED** [review]
LGTM! Nice work! Makes it a lot clearer now! There might be some follow on work, but we can address that in another PR too. 👍 🙌 🎉

**Pull request merged**